src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -128,6 +128,8 @@ private static bool PublishProjectContext(ProjectContext context, string outputP
                 PublishFiles(export.NativeLibraries, outputPath, nativeSubdirectories);
             }
 
+            CopyContents(context, outputPath);
```

Comment:
I believe we have code similar to this in compile. Should we be sharing this kind of thing between OUR commands?